docs/manual/index-checker.tex

```diff
@@ -370,6 +373,31 @@ \section{Arrays of the same length\label{index-samelen}}
   \code{null} has this type.
 \end{description}
 
+\section{Special handling for binarySearch\label{index-searchindex}}
+
+The JDK's implementation of
+\sunjavadoc{java/util/Arrays.html\#binarySearch-java.lang.Object:A-int-int-java.lang.Object-}{binary search}
+uses a contract that is not directly
```

Comment:
It *is* supported, because the Search Index Checker is part of the Index Checker.